docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
+
+.. py:function:: config(length=32, queue=3, channel=7, power=0, address=b'uBit\x00', data_rate=radio.MODE_1MHZ)
+
+    Configures the various settings relating to the radio. The specified
+    default values are sensible.
+
+    The ``length`` defines the size, in bytes, of a message sent via the radio.
+    It can be up to 256 (????? CHECK THIS from memory) bytes long.
+
+    The ``queue`` specifies the number of messages that can be stored on the
+    incoming message queue.
+
+    The ``channel`` can be an integer value between 0 and 100 (inclusive) that
```

Comment:
I might be wrong about the following comment, since English is not my first language, but `between 0 and 100 (inclusive)` to me sounds like it includes 100 but not 0, is it more clear to say "both inclusive"? or maybe `from 0 to 100 (inclusive)`.